test_dbNodeFromNode uses stale expected value after dbNodeFromCode tag fix in #822

### What is the issue?

PR #822 corrected dbNodeFromCode to join code.tags correctly, producing "111-111,222-222" for input tags ["111-111", "222-222"]. The expected value in test_dbNodeFromNode was never updated and still expects "1,2", so the test always fails.

### Expected Behaviour

test_dbNodeFromNode should pass and correctly validate that dbNodeFromCode produces a comma-separated string of tag values.

### Actual Behaviour

AssertionError: '111-111,222-222' != '1,2'

### Steps to reproduce

1. Run the test suite: flask test
2. Observe the failure on test_dbNodeFromNode in db_test.py

--- a/application/tests/db_test.py
+++ b/application/tests/db_test.py
@@ -1092,7 +1092,7 @@ class TestDB(unittest.TestCase):
                 name="ccc",
                 description="c2",
                 link="https://example.com/code/hyperlink",
-                tags="1,2",
+                tags="111-111,222-222",
                 ntype=defs.Credoctypes.Code.value,
             ),
         }

